tests/web3js/debug_traces_test.js

```diff
@@ -283,15 +283,15 @@ it('should retrieve transaction traces', async () => {
         txTrace,
         {
             from: conf.eoa.address.toLowerCase(),
-            gas: '0xb50d',
+            gas: '0xb56b',
             gasUsed: '0x6147',
             to: contractAddress.toLowerCase(),
             input: '0xc550f90f',
             output: '0x0000000000000000000000000000000000000000000000000000000000000006',
             calls: [
                 {
                     from: contractAddress.toLowerCase(),
-                    gas: '0x54e1',
+                    gas: '0x553d',
```

Comment:
Change comes from: `EIP-7623: Increase calldata cost`.
See: https://mixbytes.io/blog/the-prague-electra-pectra-hardfork-explained#rec860520137 .